src/map/pc.h

```diff
@@ -482,6 +482,7 @@ END_ZEROED_BLOCK;
 	int change_level_3rd; // job level when changing from 2nd to 3rd class [jobchange_level_3rd in global_reg_value]
 
 	char fakename[NAME_LENGTH]; // fake names [Valaris]
+	int fakename_option; // @fakename display options [Wolfie]
```

Comment:
fakename_option by default 0, this mean hide all and fakename is empty. Probably it should be 3 by default?
If yes set fakename value in pc_setnewpc